packages/react/src/Label/Label.module.css

```diff
@@ -1,10 +1,24 @@
 .Label {
-  --brand-Label-gradient: radial-gradient(
```

Comment:
I was going to ask whether you think we should continue to support this CSS variable, but I've had a look in github/github and I can't see it used, so it's probably alright to remove.

Might be worth calling it out in the changelog though, wdyt?

Reply:
Hmm, these CSS variables are actually internal to the component and not meant for external use. Perhaps we should consider a different naming convention for them?

Reply:
Ah gotcha, disregard this suggestion in that case 🙂